packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/Payment/model.tsx

```diff
@@ -112,9 +112,8 @@ export const renderBank = (
   <>
     <DisplayValue>{renderBankText(value)}</DisplayValue>
     <DisplayTitle>
-      {`${value.details?.bankAccountType.toLowerCase()} account ${
-        value.details?.accountNumber
-      }`}
+      {`${value.details?.bankAccountType?.toLowerCase() || ''} account ${value
+        .details?.accountNumber || ''}`}
```

Comment:
I put fall back to empty so we have it like. I do expect with real data to have all of those set properly anyhow this will prevent worse case scenario.  
![image](https://user-images.githubusercontent.com/67264242/112850047-f210eb80-90a9-11eb-8b48-dcdfb35febaa.png)

Reply:
> I put fall back to empty so we have it like. I do expect with real data to have all of those set properly anyhow this will prevent worse case scenario.  
> 
> ![image](https://user-images.githubusercontent.com/67264242/112850047-f210eb80-90a9-11eb-8b48-dcdfb35febaa.png)
> 
>